manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
+PackageVersion: v2.8
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  InstallerUrl: https://github.com/Muhammed87643/Friday-Night-Funkin-Silent-Installer/releases/download/v1.0/Funkin.msi
+  InstallerSha256: D8EBF64024A809BA9782DC9E6CF53705231789167E7252B20934108FEE9B0E8D
+  ProductCode: '{474F6928-009E-44A0-A22A-4F0CF7DD6BE5}'
+ManifestType: installer
+ManifestVersion: 1.0.0
```

```suggestion
ManifestVersion: 1.4.0
```
Since changes are needed anyways